docs/architecture/concepts/psu-device-redirection.md

```diff
@@ -73,54 +67,56 @@ In the case of __AUTOMATIC_REDIRECTION__, information are transported as URL par
+- Alice: the honest person owner of an account with the IDP,
+- Bob: the malicious person trying to gain control on the identity of Alice,
+- UI-RP-Alice: The user agent used by Alice to Access the relying party server application (APP-RP),
+- UI-IDP-Alice: The user agent used by Alice to Access the server application of the identity provider (APP-IDP),
+- UI-RP-Bob: The user agent used by Bob to Access the relying party server application (APP-RP).
 
-Bellow is a non exhaustive list of topics of concern while using automatic redirection to collect the Alice's authorization.
+Bellow is a non exhaustive list of topics of concern while using automatic redirection to collect the Alice's authorization. A mor eextensive list can be found in [draft-ietf-oauth-security-topics-14](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-14).
 
 ### Session Fixation Attack
-A well known attack is having a Bob start a session with the relying party (UI-RP-Bob -> APP-RP) and trick Alice to use the redirect url to start an authorization session with the identity provider (UI-IDP-Alice -> APP-IDP).
+A well known attack is having Bob start a session with the relying party (UI-RP-Bob -> APP-RP) and trick Alice to use the redirect url to start an authorization session with the identity provider (UI-IDP-Alice -> APP-IDP).
 
-Some framework will allow UI-RP-Bob to simply poll for completion of the authorization request and proceed forward when Alice has provided her consent (UI-IDP-Alice -consent-> APP-IDP). This is an obvious flaw in the authorization flow as Bob will gain control to the session as soon as Alice provide her authorization to the IDP.
+Some framework will allow UI-RP-Bob to simply poll for completion of an authorization request and proceed forward when Alice has provided her consent (UI-IDP-Alice -consent-> APP-IDP). This is an obvious flaw in the authorization flow as Bob will gain control of the session as soon as Alice provides her authorization to the IDP.
 
-In order to make sure authorization being performed by Alice (UI-IDP-Alice -> APP-IDP) was really initiated by Alice (UI-RP-Alice -> APP-RP) and not by Bob (not[UI-RP-Bob -> APP-RP]), APP-IDP has to auto redirects control to APP-RP through Alice UIs (APP-IDP -> UI-IDP-Alice -> UI-RP-Alice -> APP-RP). This redirection must carry an authorization code that is used by the relying party to collect the token from the identity provider in the back channel( APP-RP -authCode-> APP-IDP). 
+In order to make sure authorization being performed by Alice (UI-IDP-Alice -> APP-IDP) was really initiated by Alice (UI-RP-Alice -> APP-RP) and not by Bob (not[UI-RP-Bob -> APP-RP]), APP-IDP has to auto redirect control to APP-RP through Alice UIs (APP-IDP -> UI-IDP-Alice -> UI-RP-Alice -> APP-RP). This redirection must carry an authorization code that is used by the relying party to collect the token from the identity provider in the back channel( APP-RP -authCode-> APP-IDP). 
```

Comment:
`has to auto redirect control` can be replaced with `has to redirect control automatically`